.github/workflows/ci.yml

```diff
@@ -1,12 +1,11 @@
 name: CI
 
 on:
-  workflow_dispatch: {}
```

Comment:
Do we want to keep this in case we need to manually publish?

Reply:
Good idea.